src/service/data_products/taxa_count.py

```diff
@@ -234,9 +243,43 @@ async def get_taxa_counts(
         await _add_subset_data_in_place(q, store, collection_id, load_ver, rank, dp_proc)
         # For now always sort by the std data. See if ppl want sort by match/selection data
         # before implementing something more sophisticated.
+    _sort_taxa_counts(q)
+
     return _taxa_counts(dp_match=dp_match, dp_sel=dp_sel, data=q)
 
 
+def _sort_dict_list(
+        dict_list: list[dict],
+        key: str,
+        reverse: bool = True):
```

Comment:
It looks like this is never used by calling code and I don't see a use case for it - is it necessary?

Reply:
removed.

Reply:
:+1: